src/Sulu/Component/Content/Metadata/Parser/SchemaXmlParser.php

```diff
@@ -24,13 +24,13 @@ public function load(\DOMXPath $xpath, \DOMNode $contextNode)
         $allOfNode = $xpath->query('x:allOf', $contextNode)->item(0);
         $allOfs = [];
         if ($allOfNode) {
-            $allOfs = $this->loadAllOfs($xpath, $allOfNode, $contextNode);
+            $allOfs = $this->loadAllOfs($xpath, $allOfNode);
         }
 
         $anyOfNode = $xpath->query('x:anyOf', $contextNode)->item(0);
         $anyOfs = [];
         if ($anyOfNode) {
-            $anyOfs = $this->loadAnyOfs($xpath, $anyOfNode, $contextNode);
+            $anyOfs = $this->loadAnyOfs($xpath, $anyOfNode);
```

Comment:
not sure here as the params are: `(\DOMXPath $xpath, \DOMNode $contextNode)`  @danrot

Reply:
I think this change is correct as well